searx/utils.py

```diff
@@ -465,3 +465,7 @@ def get_xpath(xpath_str):
 def eval_xpath(element, xpath_str):
     xpath = get_xpath(xpath_str)
     return xpath(element)
+
+
+def str_split(string, delimiter, count):
+    return string.split(delimiter, maxsplit=count)
```

Comment:
Why not use string.split directly ?

Reply:
It is a leftover. I've prepared the PR a few weeks ago and I forgot that we have dropped Python 2. :D I will fix it. Thanks for spotting the problem.